pkg/compiler/codegen.go

```diff
@@ -207,6 +207,10 @@ func (c *codegen) emitLoadVar(name string) {
 
 // emitStoreVar stores top value from the evaluation stack in the specified variable.
 func (c *codegen) emitStoreVar(name string) {
+	if name == "_" {
+		emit.Opcode(c.prog.BinWriter, opcode.DROP)
```

Comment:
It's fine in terms of correctness (that we care about more at the moment), but it's suboptimal. Consider common use cases like
```
for _, v := range smth {
}
```
You would load and drop a key for each iteration when you can not do that at all. Though maybe there are not a lot of cases like this and we can tweak them later.